### PR TITLE
Optionalparams

### PR DESCRIPTION
**Pull request by corpaul**
made some libswift params optional

### CODE REVIEW

File: experiments/libswift/run_experiment.sh [hunk 1]
```diff
@@ -120,25 +120,33 @@ else
 	HETEROGENEOUS_OFFSET=false
 fi
 
-# @CONF_OPTION LEECHER_TIME: Time a leecher will remain running, 
+# @CONF_OPTION LEECHER_TIME: Time a leecher will remain running (optional),
```
Comment (whirm) [inline experiments/libswift/run_experiment.sh]: What's the default behavior?

Comment (corpaul) [inline experiments/libswift/run_experiment.sh]: default = libswift behavior = it exits when it is done downloading

Comment (whirm) [inline experiments/libswift/run_experiment.sh]: I mean, if it's optional, you should document the default behaviour.

**Comment by whirm** [comment]
Is it ready to pull now?

**Comment by corpaul** [comment]
yes

**Pull request merged**